xoinvader/scoreboard.py

```diff
@@ -0,0 +1,80 @@
+"""Scoreboard routines.
+
+Scoreboard file is CSV of username,score rows.
+If file doesn't exists it will be recreated from default stub.
```

Comment:
exists -> exist, don't need 's' if there's does already

Reply:
Ah, I know it, typo.